app/components/Tests/DashboardSettingsCard.test.tsx

```diff
@@ -0,0 +1,81 @@
+import { render, fireEvent } from "@testing-library/react-native"
+import { TamaguiProvider } from "tamagui"
+
+import { DashboardSettingsCard } from "@/components/DashboardSettingsCard"
+import config from "@/tamagui.config"
+import { ThemeProvider } from "@/theme/context"
+
+jest.mock("@/components/Icon", () => ({
+  Icon: ({ icon, testID }: any) => {
+    const MockIcon = require("react-native").Text
+    return <MockIcon testID={testID}>{`Icon: ${icon}`}</MockIcon>
+  },
+}))
+
+jest.mock("@/components/Text", () => ({
+  Text: ({ children, style }: any) => {
+    const MockText = require("react-native").Text
+    return (
+      <MockText style={style} testID="mock-text">
+        {children}
+      </MockText>
+    )
+  },
+}))
+
+const TestWrapper = ({ children }: { children: React.ReactNode }) => (
+  <TamaguiProvider config={config} disableInjectCSS>
+    <ThemeProvider>{children}</ThemeProvider>
+  </TamaguiProvider>
+)
+
+describe("DashboardSettingsCard", () => {
+  const defaultProps = {
+    title: "Test Card",
+    subtitle: "Test Subtitle",
+    checked: false,
+    onToggle: jest.fn(),
+    onDrag: jest.fn(),
+  }
+
+  const renderComponent = (props = {}) => {
+    const finalProps = { ...defaultProps, ...props }
+    return render(
+      <TestWrapper>
+        <DashboardSettingsCard {...finalProps} />
+      </TestWrapper>,
+    )
+  }
+
+  beforeEach(() => {
+    jest.clearAllMocks()
+  })
+
+  it("renders title and subtitle", () => {
+    const { getByText } = renderComponent()
+    expect(getByText("Test Card")).toBeTruthy()
+    expect(getByText("Test Subtitle")).toBeTruthy()
+  })
```

Comment:
Let's specifically test that the title and subtitle are rendered instead of both texts just being on the component. Add in the `testID` into the `DashboardSettingCard` and include `testID` as a prop in your mocked `Text` component above

```suggestion
  it("renders title and subtitle", () => {
    const { getByTestId } = renderComponent()
    expect(getByTestId("settings-card-title").props.children).toBe("Test Card")
    expect(getByTestId("settings-card-subtitle").props.children).toBe("Test Subtitle")
  })
```